applications/GeoMechanicsApplication/tests/cpp_tests/test_steady_state_Pw_piping_element.cpp

```diff
@@ -79,7 +80,10 @@ namespace Kratos::Testing
 
             // Create the test piping element
             std::vector<ModelPart::IndexType> element_nodes{1, 2, 3, 4};
```

Comment:
`element_nodes` appears to be no longer in use, so please remove it.

Reply:
Good catch, removed!